Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Views/AssetBrowserTreeView.cpp

```diff
@@ -539,15 +539,90 @@ namespace AzToolsFramework
             }
         }
 
+        FixedSizeMessageBox::FixedSizeMessageBox(
+            const QString& title,
+            const QString& text,
+            const QString& informativeText,
+            const QString& detailedText,
+            QMessageBox::Icon icon,
+            QMessageBox::StandardButton standardButton,
+            QMessageBox::StandardButton defaultButton,
+            QWidget* parent)
+            : QMessageBox(parent)
+        {
+            setWindowTitle(title);
+            setText(text);
+            setInformativeText(informativeText);
+            setDetailedText(detailedText);
+            setIcon(icon);
+            setStandardButtons(standardButton);
+            setDefaultButton(defaultButton);
+        }
+
+        void FixedSizeMessageBox::SetSize(int width, int height)
+        {
+            QSpacerItem* horizontalSpacer = new QSpacerItem(width, height, QSizePolicy::Minimum, QSizePolicy::Expanding);
+            QGridLayout* layout = (QGridLayout*)this->layout();
+            layout->addItem(horizontalSpacer, layout->rowCount(), 0, 1, layout->columnCount());
+        }
+
```

Comment:
Because this is supposed to be a fixed size message box, the size should be passed into the constructor instead of a separate function. That will also make it safer and prevent somebody from calling set size more than once, which will repeatedly add spacers to the layout.

```suggestion
        FixedSizeMessageBox::FixedSizeMessageBox(
            const QString& title,
            const QString& text,
            const QString& informativeText,
            const QString& detailedText,
            QMessageBox::Icon icon,
            QMessageBox::StandardButton standardButton,
            QMessageBox::StandardButton defaultButton,
            const int width, 
            const int height,
            QWidget* parent)
            : QMessageBox(parent)
        {
            setWindowTitle(title);
            setText(text);
            setInformativeText(informativeText);
            setDetailedText(detailedText);
            setIcon(icon);
            setStandardButtons(standardButton);
            setDefaultButton(defaultButton);

            QSpacerItem* horizontalSpacer = new QSpacerItem(width, height, QSizePolicy::Minimum, QSizePolicy::Expanding);
            QGridLayout* layout = (QGridLayout*)this->layout();
            layout->addItem(horizontalSpacer, layout->rowCount(), 0, 1, layout->columnCount());
        }

```

Reply:
Fixed.